hack/jenkins/e2e.sh

```diff
@@ -964,12 +966,18 @@ export PATH=${PATH}:/usr/local/go/bin
 export KUBE_SKIP_UPDATE=y
 export KUBE_SKIP_CONFIRMATIONS=y
 
+# Kubemark
+export USE_KUBEMARK="${USE_KUBEMARK:-false}"
```

Comment:
Kubemark only exists on master right now, right?

This makes me realize that this practice isn't fully sustainable - if we add more config vars in the future (likely), we shouldn't have to backport default values for those to each of the release branches.